docker/Dockerfile.api

```diff
@@ -2,6 +2,14 @@ ARG base_image_tag
 
 FROM deepset/haystack:${base_image_tag}
 
+ENV SERVICE_NAME="gunicorn-service"
+
+RUN addgroup --gid 1001 $SERVICE_NAME && \
+    adduser --gid 1001 --uid 1001 --shell /bin/false --disabled-password $SERVICE_NAME && \
+    mkdir -p /var/log/$SERVICE_NAME && \
+    chown $SERVICE_NAME:$SERVICE_NAME /var/log/$SERVICE_NAME
+
+
 # Create a folder for the /file-upload API endpoint with write permissions
 RUN mkdir -p /opt/file-upload && chmod 777 /opt/file-upload
```

Comment:
Should we change the owner of `/opt/file-upload` here to `$SERVICE_NAME`? Also, `chmod 777` seems to be very permissive, should we restrict the permissions a bit?

Reply:
Very good point @bogdankostic . I'll fix this, try it out and update this PR.